src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -208,12 +215,19 @@ object EasyIngest {
       System.exit(13)
     }
 
+  private class CompositeException(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)} ${getStackTrace(t)}"))
+  private class CompositeExceptionMessages(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)}"))
```

Comment:
I don't like the name of this exception class. IMHO all exception classes should have a postfix `Exception` (or `Error`). Maybe something like `CompositeMessageException`?